client/src/components/ChangeAround.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import Nav from './Nav';
+import '../App.css';
+import { useState, useEffect } from 'react';
+
+
+//allows you to edit a page, changes the url of the page (to each id of each specific item in the database, sets which entry is on the page, then sets the placeholder text for each item in the editable form to be the text that resides in the database for each journal entry.)
+
+//tried to get the delete and edit buttons to work but the proxy error is preventing it from doing so. googling did not help me solve. 
+export default function ChangeAround(props) {
+    let entryId = window.location.pathname.replace('change/','')
+    console.log(entryId)
+    const [entry, setEntry] = useState([]);
+    
+    useEffect(() => {
+        if (entry.length === 0)  {
+            fetch(`/api/`+entryId) 
```

Comment:
From what I can see, the proxy error is coming from the extra `/` at the end of the `/api/`